docs/ods-configuration.adoc

```diff
@@ -47,47 +52,58 @@ Alternatively to referencing tasks, ad-hoc task definitions can also be specifie
 [source,yaml]
 ----
 pipeline:
-  tasks:
-  - name: build
-    taskSpec:
-      steps:
-        - name: message
-          image: ubuntu
-          script: |
-            #!/usr/bin/env bash
-            echo "hello world"
-    workspaces:
-    - name: source
-      workspace: shared-workspace
+  - tasks:
+    - name: build
+      taskSpec:
+        steps:
+          - name: message
+            image: ubuntu
+            script: |
+              #!/usr/bin/env bash
+              echo "hello world"
+      workspaces:
+      - name: source
+        workspace: shared-workspace
 ----
 
-The pipeline created based on the configuration will have a workspace named `shared-workspace` available, which is backed by a PVC (named `ods-pipeline`) in your namespace.
+The pipelines created based on the configuration will have a workspace named `shared-workspace` available, which is backed by a PVC (named `ods-pipeline`) in your namespace.
 
 Next to the tasks you specify, `ods-pipeline` will automatically inject two tasks into the pipeline, `ods-start` and `ods-finish`. `ods-start` is inserted as the very first task, checking out the repository given in the webhook, setting the Bitbucket build status and dealing with Nexus artifacts, etc.
 
 The `ods-finish` task is added as a final task to the pipeline. Final tasks run at the end, regardless whether all previous tasks succeeded. The `ods-finish` sets the Bitbucket build status and deals with Nexus artifacts, etc.
 
+Using the `trigger` section, different pipelines may be configured and will be run based on the Bitbucket webhook event received. To further narrow down pipeline selection, the following criteria may be specified:
+
+`event`:: List of events that trigger the pipeline. Patterns as supported by link:https://pkg.go.dev/path#Match[`path.Match`] may be used to match events.
+`branches`:: List of branches to which the triggering event shall refer. Patterns as supported by link:https://pkg.go.dev/path#Match[`path.Match`] may be used to match branches.
+`exceptBranches`:: List of branches to which the triggering event may not refer. Patterns as supported by link:https://pkg.go.dev/path#Match[`path.Match`] may be used to match the excluded branches.
+`prComment`:: Define a prefix a comment has to start with. Might be used to implement functionality like slash commands.
+
+Currently, the Bitbucket events `repo:refs_changed` (fired on push to a Bitbucket repository) and any Pull Request related events (event types with prefix `pr:`) are supported (for a full list of events, please refer to the link:https://confluence.atlassian.com/bitbucketserver/event-payload-938025882.html[Atlassian Bitbucket Documentation]). Only the first pipeline matching all conditions in the trigger section will be executed. If no trigger section is specified, the pipeline will always match.
```

Comment:
Great, like it 👍 

Since I spotted a typo further down so a small change has to be made anyway, let me add more small nagging here ;) but feel free to also leave as is ...

* "will be run based on the Bitbucket webhook event received" -> what happens if no even is specified? maybe we can reword the sentence of make that clear in the `event` section
* "Using the `trigger` section, different pipelines may be configured" is maybe not 100%exact, maybe we could say sth. along the lines of "multiple pipelines can be configured, and the web hook data is matched against the config in the trigger section, selecting the pipeline to run"?